src/formatDate.js

```diff
@@ -50,7 +50,34 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const typeFormat = toFormat.slice(0, 3);
+  const symbolFromFormat = fromFormat[3];
+  const symbolTOFormat = toFormat[3];
```

Comment:
```suggestion
  const symbolToFormat = toFormat[3];
```
можливо назва змінної не найкраща, ментори, коли переглянуть скажуть точніше